TFLite/src/TFLiteParser.py

```diff
@@ -0,0 +1,142 @@
+import flatbuffers
+import inspect
+from tflite import BuiltinOperator
+from tflite import Model
+from common import Node
+from common import Edge
+from common import Graph
+import OpToNode
+import TensorToEdge
+
```

Comment:
import ordering incorrect. Complete module path with package name is considered when lexicographically sorting.

Reply:
Ignore this. Same as the first PR. Unless the other PR is merge there will be these files shows.